group_project_v2/mixins.py

```diff
@@ -160,22 +317,21 @@ def workgroup(self):
         workgroup = self._get_workgroup(self.project_api, self.user_id, self.course_id)
         return workgroup if workgroup else self.FALLBACK_WORKGROUP
 
-    @staticmethod
-    @memoize_with_expiration()
-    def _get_workgroup(project_api, user_id, course_id):
+    # COMMENT: This was memoized and static, now it is not. Rationale is that:
+    # All API calls are memoized so this is not a timesaver, and we need
+    # settings here --- and settings rely on settings service.
+    def _get_workgroup(self, project_api, user_id, course_id):
```

Comment:
This was memoized and static, now it is not. Rationale is that:

*  All API calls are memoized so this is not a timesaver, 
* we need settings here --- and settings rely on settings service, which require this to be an instance method